run-build.ps1

```diff
@@ -114,6 +114,7 @@ if ($NoBuild)
 }
 else
 {
+    dotnet msbuild build.proj /t:WriteDynamicPropsToStaticPropsFiles
```

Comment:
Can we give this target a better name? One that will be easier to type if we want to run this without the full build?